examples/standalone/benchmarks/plotting.py

```diff
@@ -30,40 +30,80 @@
         key=lambda k: datetime.strptime(k["setup"]["timestamp"], "%d/%m/%Y %H:%M:%S")
     )
     for plottype in ["mainLoop", "initializationTotal"]:
-        keyval = (
-            ["main_loop"] if plottype == "mainLoop" else ["initialization", "total"]
+        data = (
```

Comment:
Would it make sense to move this on-the-fly definition to a CONSTANT at the beginning of this script with a comment, moving in the direction of making this configurable.

Reply:
Is this `data` or `timers`?